registry/infrastructure/repositories/service_repository.py

```diff
@@ -0,0 +1,75 @@
+from datetime import datetime as dt
+from registry.domain.factory.service_factory import ServiceFactory
+from registry.infrastructure.models import Service,ServiceGroup, ServiceState, ServiceReviewHistory
+from registry.infrastructure.repositories.base_repository import BaseRepository
+from sqlalchemy import func
+
+
+class ServiceRepository(BaseRepository):
+    def get_services_for_organization(self, org_uuid, payload):
+        raw_services_data = self.session.query(Service). \
+            filter(getattr(Service, payload["search_attribute"]).like("%" + payload["search_string"] + "%")). \
+            filter(Service.org_uuid == org_uuid). \
+            order_by(getattr(getattr(Service, payload["sort_by"]), payload["order_by"])()). \
+            slice(payload["offset"], payload["limit"]).all()
+
+        services = []
+        for service in raw_services_data:
+            services.append(ServiceFactory().convert_service_db_model_to_entity_model(service).to_dict())
```

Comment:
Repository should return list of service. The method to_dict() should not be called in repository.